test/run-drun/stable-mem-nat32.mo

```diff
@@ -2,16 +2,16 @@ import P "mo:⛔";
 import StableMemory "stable-mem/StableMemory";
 actor {
 
-  stable var n : Nat32 = 0;
+  stable var n : Nat64 = 0;
   assert (n == StableMemory.size());
 
-  func valOfNat32(n : Nat32) : Nat32 { n };
-  let inc : Nat32 = 4;
+  func valOfNat64(n : Nat64) : Nat32 { P.natToNat32(P.nat64ToNat(n)) };
```

Comment:
We really need more efficient conversions...

Reply:
Yep.